### ISSUE TITLE
Explorer mode for Fullscreen mode

### ISSUE THREAD

**Issue by JosefNemec** [JosefNemec/Playnite#1278]
Should allow user to browse different field categories. For example view all available platforms, tags etc. and then navigate to games assigned to them.

**Comment by tonyconman** [comment]
It would be nice to be able to create categories in full screen mode as well as change what category a game is in. Currently you can filter by categories which you created in desktop mode but I don't see any way to actually create/edit them in full screen.

**Comment by HVR88** [comment]
Any possibility this will be added to the 9.0 milestone list?

**Comment by JosefNemec** [comment]
No, P9 milestone was finalized a long time ago and it's based on Patreon votes.

**Comment by HVR88** [comment]
Understood, maybe next time.
